shell/platform/embedder/embedder_external_view.cc

```diff
@@ -83,15 +85,28 @@ bool EmbedderExternalView::Render(const EmbedderRenderTarget& render_target) {
       << "Unnecessarily asked to render into a render target when there was "
          "nothing to render.";
 
-  auto surface = render_target.GetRenderSurface();
-  if (!surface) {
+  auto impeller_target = render_target.GetImpellerRenderTarget();
+  if (impeller_target.has_value()) {
+    auto aiks_context = render_target.GetAiksContext();
+
+    DisplayListBuilder dl_builder;
+    dl_builder.SetTransform(&surface_transformation_);
+    slice_->render_into(&dl_builder);
+
+    auto dispatcher = impeller::DlDispatcher();
```

Comment:
Aha. I've been trying to making this the default in the engine but it seems so counter to convention. My reasoning is that `auto foo = FooType{};` makes it impossible to have foo uninitialized.

Reply:
But the dl_builder above uses the other style? Just use the same style everywhere?

Reply:
Done. I also prefer `auto foo = ...`, but tend to heavily bias towards whatever's nearby.